content/en/_index.html

```diff
@@ -110,21 +110,29 @@
 	{{< card header="Source Code Control Events" >}}
 	Represent events associated with the management of source code assets
 	{{< /card >}}
-	{{< card header="Continuous Integration Pipelines Events" >}}
+	{{< card header="Continuous Integration Events" >}}
 	Represent events related to building, testing, packaging, and releasing software artifacts
 	{{< /card >}}
 {{< /cardpane >}}
 {{< cardpane >}}
-	{{< card header="Continuous Deployment Pipelines Events" >}}
+	{{< card header="Testing Events" >}}
+	Represent events associated with Testing activities
+	{{< /card >}}
+	{{< card header="Continuous Deployment Events" >}}
 	Represent events related to environments where the artifacts produced by the integration pipelines are intended to run
 	{{< /card >}}
+{{< /cardpane >}}
+{{< cardpane >}}
+	{{< card header="Continuous Operations Events" >}}
+	Represent events associated with Continuous Operations activities, such as incidents and their resolution
+	{{< /card >}}
 	{{< card header="Core Events" >}}
 	Represent shared abstractions such as Task Runs and Pipeline Runs
 	{{< /card >}}
 {{< /cardpane >}}
 
 <p>
-	There is interest in implementing in many of your favorite CI/CD tools, including ArgoCD, Flux, Jenkins, Jenkins-X, Keptn, Knative, Ortelius, Shipwright, Spinnaker, Tekton. Some of them have started proof-of-concept implementations with CDEvents.
+	CDEvents is implemented or being integrated by many of your favorite SDLC tools, including Argo, Flux, Guac, Harbor, Jenkins, Keptn, Spinnaker, Tekton, Testkube, Tracetest and more.
```

Comment:
SDLC is not a common term in my neighborhood. Do you mind spelling it out, or add a link to the definition you assume?

Reply:
Done